config/deploy/production.rb

```diff
@@ -1,61 +1,11 @@
 # server-based syntax
 # ======================
-# Defines a single server with a list of roles and multiple properties.
-# You can define all roles on a single server, or split them:
-
-# server 'example.com', user: 'deploy', roles: %w{app db web}, my_property: :my_value
-# server 'example.com', user: 'deploy', roles: %w{app web}, other_property: :other_value
-# server 'db.example.com', user: 'deploy', roles: %w{db}
-server 'signage.chapman.edu', user: 'charles', roles: %w{app db web}
-
+server 'signage.chapman.edu', user: 'charles', roles: %w{app db web postgres}
```

Comment:
You'll need to explain this new `postgres` role for me. Description in task below says:

> Clone the server database to your local machine

But this is being called for production here and staging below. So I'd be worried it was going to clone the server database somewhere everytime cap was run.